firebase-firestore/src/main/java/com/google/firebase/firestore/model/mutation/Mutation.java

```diff
@@ -162,13 +154,13 @@ static SnapshotVersion getPostMutationVersion(@Nullable MaybeDocument maybeDoc)
    * result of applying a transform) for use after a mutation containing transforms has been
    * acknowledged by the server.
    *
-   * @param maybeDoc The current state of the document after applying all previous mutations.
+   * @param mutableDocument The current state of the document after applying all previous mutations.
    * @param serverTransformResults The transform results received by the server.
    * @return The transform results list.
```

Comment:
ultranit: "list" is no longer accurate, since the return type has changed to a map. Please update. Also on line 192.

Reply:
Fixed (here and below)